modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -102,7 +102,7 @@ def get_original_status_code(error)
 
     def will_retry?(auto_claim, error)
       msg = if auto_claim.evss_response.present?
-              auto_claim.evss_response&.dig(0, 'key')
+              auto_claim.evss_response&.dig('key')
```

Comment:
With this no longer being nested inside `messages` the index is no longer required

Reply:
No, that was totally my mistake, I got the format on that wrong, thank you for catching that!

Reply:
@FonzMP On your point about the index and only checking the first one.  There was a test in here that appeared to want to test that but there was never any code implemented actually handling that in the `will_retry?` method when there were more then one errors in the array.  I would prefer to put another ticket together to handle that since it will be a slightly larger code change along with tests updates, but let me know if you feel strongly on that one and we can address it either way.

Reply:
oh, 100% agree on that going in to a different ticket/PR - that is not a change needed here, currently. It's already in play with `0` so let's do it in another. 



And I promise I don't ignore these, my github emails are just filtered. Ping me if I ever take a day to respond. - Apologies.

Reply:
Good to resolve once you've seen this. 👍